src/mainwindow.cpp

```diff
@@ -108,25 +113,34 @@ void MainWindow::logMessageOutput(QtMsgType type, const QMessageLogContext & /*c
       logFile->write(": [Fatal]    ");
       break;
   }
-  QByteArray localMsg = msg.toLocal8Bit();
+
+  QString nonConstMsg(msg);
+  if (!msg.startsWith("!"))     // Qt/system log message?
+    nonConstMsg = "123" + msg;  // "123" gets removed below
+
+  QByteArray localMsg = nonConstMsg.toLocal8Bit();
   QByteArray displayMsg = localMsg.mid(3);
+
   logFile->write(displayMsg);
   logFile->write("\n");
+
   if (type == QtFatalMsg) {
     logFile->close();
+#if defined(QT_NO_DEBUG) // allow debugger to survive fatal error
     abort();
+#elif defined (Q_OS_WIN) // TBD: non-windows? (QtCreator 10.x on Mojave is OK)
+    __debugbreak();
```

Comment:
Does __debugbreak work on Linux, Macos, and Windows?

Reply:
It worked on my intel based MacBook but not sure that means all MacOS. First hit on stack overflow has an incomplete solution: https://stackoverflow.com/questions/173618/is-there-a-portable-equivalent-to-debugbreak-debugbreak
... not sure where to go with this.